android/library/src/test/java/io/logto/android/auth/browser/BrowserSignOutFlowTest.kt

```diff
@@ -15,50 +15,48 @@ import io.logto.client.exception.LogtoException.Companion.INVALID_REDIRECT_URI
 import io.logto.client.exception.LogtoException.Companion.SIGN_OUT_FAILED
 import io.logto.client.exception.LogtoException.Companion.USER_CANCELED
 import io.logto.client.model.OidcConfiguration
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.spyk
+import io.mockk.verify
 import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.Job
 import kotlinx.coroutines.ObsoleteCoroutinesApi
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.mockito.Mock
-import org.mockito.Mockito.`when`
-import org.mockito.Mockito.spy
-import org.mockito.MockitoAnnotations
-import org.mockito.kotlin.anyOrNull
-import org.mockito.kotlin.argumentCaptor
-import org.mockito.kotlin.doAnswer
-import org.mockito.kotlin.mock
-import org.mockito.kotlin.verify
 import org.robolectric.Robolectric
 import org.robolectric.RobolectricTestRunner
 
 @RunWith(RobolectricTestRunner::class)
 @ObsoleteCoroutinesApi
 @ExperimentalCoroutinesApi
 class BrowserSignOutFlowTest {
-    @Mock
+    @RelaxedMockK
     private lateinit var logtoConfigMock: LogtoConfig
 
-    @Mock
+    @RelaxedMockK
     private lateinit var logtoAndroidClientMock: LogtoAndroidClient
 
-    @Mock
+    @RelaxedMockK
     private lateinit var dummyOidcConfiguration: OidcConfiguration
 
-    @Mock
+    @RelaxedMockK
     private lateinit var onCompleteMock: HandleLogtoExceptionCallback
 
     private lateinit var browserSignOutFlow: BrowserSignOutFlow
 
-    private val logtoExceptionCaptor = argumentCaptor<LogtoException>()
+    private val logtoExceptionSlot = slot<LogtoException>()
```

Comment:
`onCompleteMock` receives a nullable value as it's parameter.
So our `logtoExceptionSlot` should support capturing nullable value